### PR TITLE
bootstrap theme is required for building the help page

### PR DESCRIPTION
**Pull request by josenavas**
in a clean environment, the sphinx-bootstrap-theme doesn't get installed and it is required in order to build the documentation, which is one of the steps of  `qiita pet webserver start`

### CODE REVIEW

**Comment by antgonza** [comment]
:+1: if tests pass.

File: setup.py [hunk 1]
```diff
@@ -91,13 +91,14 @@
                         ]},
       scripts=glob('scripts/*'),
       extras_require={'test': ["nose >= 0.10.1", "pep8", 'mock'],
-                      'doc': ["Sphinx >= 1.2.2", "sphinx-bootstrap-theme"]},
+                      'doc': ["Sphinx >= 1.2.2"]},
```
Comment (ElDeveloper) [inline setup.py]: Can you move Sphinx to the `install_requires` section?

Comment (josenavas) [inline setup.py]: Done

Comment (ElDeveloper) [inline setup.py]: Great, once tests pass this is ready for merge!

**Comment by josenavas** [comment]
@ElDeveloper tests are passing!

**Pull request merged**